diff --git a/drivers/kernelsu/policy/app_profile.c b/drivers/kernelsu/policy/app_profile.c
--- a/drivers/kernelsu/policy/app_profile.c
+++ b/drivers/kernelsu/policy/app_profile.c
@@ -216,6 +216,9 @@
     setup_selinux(profile.selinux_domain, cred);
 
 	commit_creds(cred);
+#ifdef CONFIG_KSU_SUSFS
+	current->susfs_task_state = 0;
+#endif
 
 	disable_seccomp();
 
@@ -248,4 +251,7 @@
 
     setup_selinux(KERNEL_SU_CONTEXT, cred);
     commit_creds(cred);
+#ifdef CONFIG_KSU_SUSFS
+    current->susfs_task_state = 0;
+#endif
 }
